test/ooyala/agent_test.js

```diff
@@ -39,3 +45,35 @@ test('An Ooyala agent can fetch videos for an organization', function (t) {
 		})
 		.catch(t.end);
 });
+
+test('An Ooyala agent can fetch videos for an organization filtered by labels', function (t) {
+	t.plan(1);
+
+	nock('https://api.ooyala.com').get('/v2/publishing_rules/22222').times(2).query(true).reply(200, mockPublishingRule);
+	nock('https://api.ooyala.com')
+		.filteringPath(/expires=[^&]*&signature=[^&]*/g, 'expires=XXX&signature=YYY')
+		.get('/v2/assets?include=metadata%2Clabels&where=asset_type%3D\'video\'%20AND%20labels%20INCLUDES%20\'Published\'&api_key=KEY&expires=XXX&signature=YYY')
+		.reply(200, mockFilteredAssets);
+	nock('https://api.ooyala.com')
+		.filteringPath(/expires=[^&]*&signature=[^&]*/g, 'expires=XXX&signature=YYY')
+		.get('/v2/assets?include=metadata%2Clabels&where=asset_type%3D\'channel\'%20OR%20asset_type%3D\'channel_set\'%20AND%20labels%20INCLUDES%20\'Published\'&api_key=KEY&expires=XXX&signature=YYY')
+		.reply(200, mockFilteredAssets);
```

Comment:
You shouldn't need to combine them. They are separate calls to different asset types.